src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -205,10 +205,9 @@ public void TrackMetric(string name, double value, IDictionary<string, string> p
         }
 
         /// <summary>
-        /// Send a <see cref="MetricTelemetry"/> for aggregation in Metric Explorer.
+        /// Send a <see cref="MetricTelemetry"/> for representing aggregated metric data.
         /// Create a separate <see cref="MetricTelemetry"/> instance for each call to <see cref="TrackMetric(MetricTelemetry)"/>.
         /// </summary>
-        [Obsolete("This method is obsolete. Use TrackAggregatedMetric method to send pre-aggregated metric data or MetricManager class to create metrics.")]
         [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
@vitalyf007 Thanks for removing `[EditorBrowseable]` from the `MetricTelemetry` class. I didn't catch that. Should we also remove it from the `TrackMetric(MetricTelemetry)` method now that it's no longer `[Obsolete]`?